lib/exercises.rb

```diff
@@ -2,19 +2,66 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n) + O(m log m)  where n = strings[] size and m = length of each word therein.
+# Space Complexity: O(n) 
 
 def grouped_anagrams(strings)
```

Comment:
The time complexity is actually O(n * m log m) where m is the length of the strings.  Since you are sorting strings n times.